prosemirror-jdita-example/docker.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
Ideally we should start from a template Dockerfile, and just template in the `${value}` bits. How hard would that be?

But... perhaps we should do it in the next PR?